x-pack/plugins/cases/public/components/create/flyout/use_cases_add_to_new_case_flyout.tsx

```diff
@@ -6,18 +6,19 @@
  */
 
 import { useCallback } from 'react';
+import { CaseAttachments } from '../../../types';
 import { useCasesToast } from '../../../common/use_cases_toast';
 import { Case } from '../../../containers/types';
 import { CasesContextStoreActionsList } from '../../cases_context/cases_context_reducer';
 import { useCasesContext } from '../../cases_context/use_cases_context';
 import { CreateCaseFlyoutProps } from './create_case_flyout';
 
-type AddToNewCaseFlyoutProps = CreateCaseFlyoutProps & {
+type AddToNewCaseFlyoutProps = Omit<CreateCaseFlyoutProps, 'attachments'> & {
   toastTitle?: string;
   toastContent?: string;
 };
 
-export const useCasesAddToNewCaseFlyout = (props: AddToNewCaseFlyoutProps) => {
+export const useCasesAddToNewCaseFlyout = (props: AddToNewCaseFlyoutProps = {}) => {
```

Comment:
Let's change the type of `props` to`props: AddToNewCaseFlyoutProps | undefined = {}` so consumers of the hook will not need to pass an empty object. For example, `useCasesAddToNewCaseFlyout({});`

Reply:
Similar to the other hook. props will default to an empty object. I fixed other pieces of the code that were passing an empty object.